launch_testing/launch_testing_examples/package.xml

```diff
@@ -13,6 +13,14 @@
   <author email="aditya.pande@openrobotics.org">Aditya Pande</author>
   <author email="sloretz@openrobotics.org">Shane Loretz</author>
 
+  <exec_depend>launch</exec_depend>
+  <exec_depend>launch_ros</exec_depend>
+  <exec_depend>launch_testing</exec_depend>
+  <exec_depend>launch_testing_ros</exec_depend>
+  <exec_depend>rclpy</exec_depend>
+  <exec_depend>rcl_interfaces</exec_depend>
+  <exec_depend>std_msgs</exec_depend>
```

Comment:
package.xml should not contain both an `exec_depend` and a `test_depend` for the same dependency; see http://wiki.ros.org/catkin/package.xml section 3.3.

In this case, it looks like all of these really should be `exec_depend`, not `test_depend`, so I'd suggest removing them from the list below.

Reply:
Thanks for enlightening me about the manifest requirement!
I've removed the `test_depends` [70e2453](https://github.com/ros2/examples/pull/357/commits/70e2453a8b8dd13951c9ba6b0ef10f253a18ebe4)